godtools/App/Features/Tools/SwiftUI/AllTools/ToolCardsView/ToolCardView/ViewModels/ToolCardViewModel.swift

```diff
@@ -96,7 +96,7 @@ extension ToolCardViewModel {
     }
     
     private func setupPublishedProperties() {
-        isFavorited = favoritedResourcesCache.isFavorited(resourceId: resource.id)
+        isFavorited = getToolIsFavoritedUseCase.getToolIsFavorited(tool: resource)
```

Comment:
Hey @rachaelblue this shouldn't be needed now that you are subscribed to a publisher below in the setupBinding() method.
```
getToolIsFavoritedUseCase.getToolIsFavoritedPublisher(tool: resource)
            .receive(on: DispatchQueue.main)
            .assign(to: \.isFavorited, on: self)
            .store(in: &cancellables)
```